src/stcal/alignment/util.py

```diff
@@ -428,7 +420,7 @@ def wcsinfo_from_model(input_model: SupportsDataWithWcs):
 
 
 def compute_scale(
-    wcs: WCS,
+    wcs: gwcs.WCS,
```

Comment:
This function computes a scale at CRPIX, not the scaling transform. I suggest replacing `Compute scaling transform` with `Compute the scale at the fiducial point on the detector.`